### ISSUE TITLE
Dark/light theme button is not accessible using the keyboard: A11y_.NET 9 template_.NET 9 template_Project_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26862]
### GitHub Tags
#A11yTCS; #A11ySev1; #Win32; #WCAG2.1.1; #SH_.NET9template_MAC_Dec2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Keyboard; #MAC;#AILimited;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
MAC Version 15.2

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter.
4. Observe that Dark/light theme button is not accessible using the keyboard keys.

### Actual Result:
 Dark/light theme button is not accessible using the keyboard keys.

### Expected Result:
 Dark/light theme button should be accessible using the keyboard keys.

### User Impact:
keyboard users will not be able to access the controls and will not be able to access the other functionality.

### Attachment:

 Dark/light theme button is not accessible using the keyboard.

https://github.com/user-attachments/assets/3e1cd097-8d76-4ffb-bd30-3ff7e81a6d18

**Comment by jfversluis** [comment]
Blocked by: https://github.com/syncfusion/maui-toolkit/issues/67

**Comment by AnKushSingh05** [comment]
Issue is not fixed

**Comment by AnKushSingh05** [comment]
Issue is fixed

https://github.com/user-attachments/assets/675defb9-b037-4af1-8e43-54aa50522c41
